app/javascript/mastodon/components/icon_button.js

```diff
@@ -42,14 +43,18 @@ export default class IconButton extends React.PureComponent {
   }
 
   render () {
-    const style = {
+    let style = {
```

Comment:
You can still use a `const` here because redefining a property on an object ≠ redefining the object. (I mean it doesn't make a difference to me but)

Reply:
i like to have let there bc then it's clear we're going to change it (babel will turn it to var anyways ...)